.github/workflows/build_and_test.yaml

```diff
@@ -113,7 +113,7 @@ jobs:
         id: build
         run: |
           pushd duckdb
-          make -j8 install
+          make -j8 install CFLAGS=-Werror CXXFLAGS=-Werror
```

Comment:
Why not using it in the Makefile as well? Too annoying in dev cycles?
In this case, maybe we want a dedicated make target?

Reply:
Fair enough - if we need this we can always add it later.